soql-standalone-parser/src/main/scala/com/socrata/soql/ast/Expression.scala

```diff
@@ -293,9 +294,10 @@ case class FunctionCall(functionName: FunctionName, parameters: Seq[Expression],
         }
         (whens ++ otherwise).encloseNesting(d"CASE" flatAlt d"CASE ", d"", d"END" flatAlt d" END")
       case other =>
+        val filterd = filter.fold(Doc.empty) { f => d" FILTER(WHERE ${f.doc})" }
```

Comment:
This will produce ugly results if it decides to break inside the filter.   Something like

```
(doc"WHERE" +#+ f.doc).enclouseNesting(doc" FILTER(", Doc.empty, ")")
```
would probably be better.